lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -9,44 +9,42 @@
     &.-purple {
       background: $purple;
     }
+  }
+}
 
-    .-white {
-      background: $white;
-    }
-
-    > .wrapper {
-      @include span(16);
-
-      @include media($tablet) {
-        @include span(12 no-gutters);
-        @include push(2);
-      }
+//@TODO - move into /patterns/card.scss
+.card {
+  background: $white;
+  margin: 0 $base-spacing / 2;
 
-      .container {
-        @include media($tablet) {
-          display: table;
-        }
-      }
+  @include media($tablet) {
+    display: table;
+  }
 
-      .container__block.-half {
-        padding: $base-spacing;
+  .card__column {
+    padding: $base-spacing;
 
-        @include media($tablet) {
-          float: none;
-          display: table-cell;
-          vertical-align: top;
-        }
+    @include media($tablet) {
+      @include span(6 of 12);
+      float: none;
+      display: table-cell;
+      vertical-align: top;
+      padding: $base-spacing;
+    }
 
-        &.-border {
-          border-top: 2px solid $light-gray;
+    + .card__column {
```

Comment:
@weerd no, tried it both ways, didn't seem necessary. Also, it's similar to something like this. https://github.com/DoSomething/dosomething/blob/v0.4.27/lib/themes/dosomething/paraneue_dosomething/scss/patterns/_promotion.scss#L89-L94